handlers/features.go

```diff
@@ -93,3 +93,73 @@ func (oh *featureHandler) GetFeatureByUuid(w http.ResponseWriter, r *http.Reques
 	w.WriteHeader(http.StatusOK)
 	json.NewEncoder(w).Encode(workspaceFeature)
 }
+
+func (oh *featureHandler) CreateOrEditStory(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	pubKeyFromAuth, _ := ctx.Value(auth.ContextKey).(string)
+	if pubKeyFromAuth == "" {
+		fmt.Println("no pubkey from auth")
+		w.WriteHeader(http.StatusUnauthorized)
+		return
+	}
+
+	newStory := db.FeatureStory{}
+	decoder := json.NewDecoder(r.Body)
+	err := decoder.Decode(&newStory)
```

Comment:
@MahtabBukhari you didn't specify the Story Uuid which will make the code always update when you try to create more than one phase, and also didn't account for UpdatedBY, there should be a conditional to check if CreatedBY exists, if it exists the UpdatedBy should be updated UpdatedBy = &now.

Reply:
> @MahtabBukhari you didn't specify the Story Uuid which will make the code always update when you try to create more than one phase, and also didn't account for UpdatedBY, there should be a conditional to check if CreatedBY exists, if it exists the UpdatedBy should be updated UpdatedBy = &now.

Hi @elraphty, Could you please review it
![image](https://github.com/stakwork/sphinx-tribes/assets/100023456/8b376508-8de9-4793-aa69-1c8029446f7e)

Reply:
I think createdBy will always be nil, because you are not fetching it from the DB

Reply:
> I think createdBy will always be nil, because you are not fetching it from the DB

Please review again:
![image](https://github.com/stakwork/sphinx-tribes/assets/100023456/972b33ae-efaf-40ad-88b5-a5cffaa26aea)